tests/antithesis/config/docker-compose-1-node.yml

```diff
@@ -9,7 +9,7 @@ services:
     group_add:
       - '${GROUP_ID:-root}'
     volumes:
-      - ${ETCD_ROBUSTNESS_DATA_PATH:-/tmp/etcddata}0:/var/etcddata0
+      - ${ETCD_ROBUSTNESS_LOCAL_DATA_PATH:-/tmp/etcddata}0:/var/etcddata0
```

Comment:
What's the value of this rename? This only complicates the review. Please move it to separate PR.